File: problemas/1000/1011.dart
import 'dart:io';
import 'dart:math';

void main() {
  final raio = double.parse(stdin.readLineSync());
  final volume = 4 / 3 * 3.14159 * pow(raio, 3);
  stdout.writeln('VOLUME = ${volume.toStringAsFixed(3)}');
}

File: problemas/1000/1011.cs
using System;

namespace uri1011
{
    internal static class Program
    {
        private static void Main()
        {
            double raio = double.Parse(Console.ReadLine());
            double volume = 4.0 / 3.0 * 3.14159 * Math.Pow(raio, 3);
            Console.WriteLine($"VOLUME = {volume:F3}");
        }
    }
}